### ISSUE TITLE
Add ability to specify `target="_blank"` in BannerText CTA

### ISSUE THREAD

**Issue by katrillion** [vtex/faststore#2097]
**Describe the feature you'd like**
Currently the link in a banner doesn't have the option to open in a new tab. I'd like to add a optional `targetBlank` property to the  link field in [BannerText schema](https://github.com/vtex/faststore/blob/65a601dcf517516259a7224c91fcfe4d107b8d9d/packages/core/cms/faststore/sections.json#L663), as well as any changes to `BannerText.tsx` and its children that are needed to apply the `target="_blank"` attribute to `LinkButton.tsx`

Example of how this could look:
```
"targetBlank": {
    "type": "boolean",
    "title": "Open link in new window?",
    "default": false
}
```
![Screenshot 2023-11-08 at 4 44 59 PM](https://github.com/vtex/faststore/assets/1152313/da5883d6-39d1-481c-ac26-7e4d00521e51)

**Describe alternatives you've considered**
It would be possible to override the BannerText section or create a custom version, but I think this change would be a good addition to the component

**Comment by hellofanny** [comment]
Hi @katrillion! Nice suggestion! 
Feel free to create a pull request with this addition 🌟 Thank you!

**Comment by hellofanny** [comment]
Thanks for the contribution 🌟 ! Updates available on @faststore/core package version `2.2.37`
